internal/pkg/aws/cloudformation/cloudformation.go

```diff
@@ -366,6 +366,18 @@ func (c *CloudFormation) ListStacksWithTags(tags map[string]string) ([]StackDesc
 	return summaries, nil
 }
 
+// CancelUpdateStack attempts to cancel the update for a CloudFormation stack specified by the stackName.
+// Returns an error if failed top cancel CloudFormation stack update.
+func (c *CloudFormation) CancelUpdateStack(stackName string) error {
+	_, err := c.client.CancelUpdateStack(&cloudformation.CancelUpdateStackInput{
```

Comment:
> You can cancel only stacks that are in the UPDATE_IN_PROGRESS state.

Should we ignore `ErrStackNotFound` and `ErrStackUpdateNotInProgress` (not an existing custom error type) by default in this function?

Reply:
yeah we can ignore that error. Since api call only cancels stack which is in UPDATE_IN_PROGRESS state